datasets/il/mod_terrorists/il_mod_terrorists.yml

```diff
@@ -115,6 +121,20 @@ lookups:
         value: null
       - match: 16/6/67
         value: "1967-06-16"
+      - match: 23 Apr. 1968
+        value: "1968-04-23"
+      - match: 25 Jun. 1964
+        value: "1964-06-25"
+      - match: 5 Dec. 1969
```

Comment:
extract_years was probably hiding these. How about instead adding a format string for them?